aspects/aspects.go

```diff
@@ -444,9 +444,17 @@ type expandedMatch struct {
 	value interface{}
 }
 
+// maxValueDepth is the limit on a value's nestedness. Since creating a highly
```

Comment:
Nitpick/personal preference: "Creating a highly nested JSON value only requires a few bytes per level, but when recursively setting such a value, each level requires about 2Kb stack. Prevent excessive stack usage by limiting the recursion depth."